docs/dawson/usecase2.md

```diff
@@ -1,12 +1,31 @@
-# ADCIRC Use Case - Creating an ADCIRC DataSets on DesignSafe
+# ADCIRC DataSets
 
-**Clint Dawson, University of Texas at Austin**
+## Creating and Publishing ADCIRC datasets on DesignSafe
 
-**Carlos del-Castillo-Negrete, University of Texas at Austin**
+**Authors**  
+- Clint Dawson, University of Texas at Austin  
+- Carlos del-Castillo-Negrete, University of Texas at Austin  
+- Benjamin Pachev, University of Texas at Austin
 
-**Benjamin Pachev, University of Texas at Austin**
+Key Words: ADCIRC,  Storm Surge, Hind-casts, Data Curation, Data Publishing
 
-## Overview
+## Resources
+
+- [DesignSafe ADCIRC Application](https://www.designsafe-ci.org/rw/workspace/#!/ADCIRC::Simulation).
+- [Jupyter nebooks on DS Juypterhub](https://www.designsafe-ci.org/rw/workspace/#!/Jupyter::Analysis).
+- [ADCIRC Wiki](https://wiki.adcirc.org/Main_Page)
+- [ADCIRC Web Page](https://adcirc.org/)
+- [pyADCIRC GitHub Repository](https://github.com/UT-CHG/pyadcirc) - pyADCIRC documentation.
+- [Creating an ADCIRC DataSet.ipynb](https://www.designsafe-ci.org/data/browser/public/designsafe.storage.community/Use%20Case%20Products/ADCIRC)
```

Comment:
Sorry one more change, could you point to specific notebooks like these:

https://deploy-preview-45--ds-use-case-preview.netlify.app/brandenberg/apiusecases/usecase/

Here's a sample link configuration that can directly open notebooks in Jupyter:

https://jupyter.designsafe-ci.org/hub/user-redirect/notebooks/CommunityData/Use%20Case%20Products/APIs/api_background.ipynb

Reply:
I think I got them now working.